emloop/hooks/training_trace.py

```diff
@@ -0,0 +1,39 @@
+"""Module with :py:class:`TrainingTrace` class."""
+from .abstract_hook import AbstractHook
+from ..utils.yaml import yaml_to_file
+from ..constants import EL_TRACE_FILE
+from ..types import EpochData
+
+from collections import OrderedDict
+from datetime import datetime
+
+
+class TrainingTrace(AbstractHook):
```

Comment:
This has to be an integral part of every training.

Reply:
This hook is now automatically created even if it is not specified in `config.yaml`.